microservices/nris_api/backend/migrations/env.py

```diff
@@ -12,14 +13,14 @@
 # This line sets up loggers basically.
 fileConfig(config.config_file_name)
 logger = logging.getLogger('alembic.env')
+schema_name = config.get_main_option('schema_name')
 
 # add your model's MetaData object here
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-from flask import current_app
 config.set_main_option('sqlalchemy.url',
-                       current_app.config.get('SQLALCHEMY_DATABASE_URI'))
+                       current_app.config['SQLALCHEMY_DATABASE_URI'])
```

Comment:
I thought we were using .get to avoid exception throwing. Is that not a pattern being followed anymore?

Reply:
I was having issues with the config and changed some settings around. I'll see if this still works with the .get. Having an object named config on the App object and a file named config in the app module is kind of confusing... Maybe we can rename the config file to settings?

Reply:
the .get pattern was about parsing/receiving form data, and in this case. we definitely want the application to crash if `SQLALCHEMY_DATABASE_URL` isn't set in the app config.